docs/data-management.rst

```diff
@@ -130,6 +130,33 @@ Browsing and rendering
 
 #. You can revert to the original settings for an Image or Dataset. For example, using the context menu for a Dataset in the tree, select ``Rendering Settings > Set Imported and Save``.
 
+
+Manage Images in Dataset/Projects
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can organize the data in the left-hand side tree by creating new Projects, Datasets and linking the Images to those new containers.
+
+#. Use the Project |image2b| and Dataset |image2c| icons above the left hand side tree or the right-click contextual menu to create new Datasets or Projects.
+
+#. Drag and drop Images or Datasets between Datasets or Projects.
+
+#. Copy Images using the right-click context menu: 
+     - Select the images to be copied, then right-click and click ``Edit > Copy Link``
```

Comment:
Use "Images" Capital 'I' everywhere for consistency. Same below. And for Datasets etc.

Reply:
Fixed in https://github.com/ome/omero-guide-introduction/pull/5/commits/3a79119f93fce5b61fe0efa6936013c4b5ce2fe2 and https://github.com/ome/omero-guide-introduction/pull/5/commits/92550478f3889681de9d6aead7a8586ff9a6d8b3